src/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -46,7 +46,7 @@ function MoveItemProperties() {
       '      </div>',
       '      <div class="item-type-info">{{vm.light}} {{::vm.classType}} {{::vm.item.typeName}}</div>',
       '      <div ng-if="vm.item.objectives" translate-values="{ percent: vm.item.percentComplete }" translate="ItemService.PercentComplete"></div>',
-      '      <dim-item-tag ng-if="vm.item.lockable && vm.featureFlags.tagsEnabled" item="vm.item"></dim-item-tag>',
+      '      <dim-item-tag ng-if="vm.featureFlags.tagsEnabled && vm.item.lockable && !vm.item.isEngram" item="vm.item"></dim-item-tag>',
```

Comment:
`isEngram` is a function, not a property. The reason I haven't done this yet is that I don't want to call a function in an expression (it's bad for perf) - we'd need to write it into each item when they're built.